assets/scss/utilities/_vert-bar.scss

```diff
@@ -0,0 +1,31 @@
+// Vertical Bar (has-vert-bar)
+//
+// Accent element for distinguishing a block of text.
+//
+// .has-vert-bar--thin - Used in membership CTA callout on stories
+// .has-vert-bar--from-bp-s - Resets styling on mobile
+//
+// Markup: <p class="has-vert-bar {{ className }} has-text-yellow"><span class="has-text-gray-dark">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores incidunt vel rerum deserunt debitis voluptas, omnis dignissimos, doloribus saepe quae itaque! Consequuntur et eveniet fugit in veritatis quos ver incidunt.</span></p>
+//
+//
+// Styleguide 8.0.3
+//
+$vert-bar-base: px-to-rem(5px);
+$vert-bar-thin: px-to-rem(4px);
+
+.has-vert-bar {
+  border-left: $vert-bar-base solid currentColor;
+  padding-left: $size-b;
```

Comment:
I love this `.has-vert-bar` helper -- it'll definitely be useful for the story-top summary and story-bottom member CTA. Because those are both on white backgrounds, though, I think having left padding baked into it might cause some alignment issues with items above/below them. Thoughts?

Reply:
hmm I don't follow 🤔, could you expand on that one? If it helps, current color relates to the text color - not bg, but I don't think that has anything to do with what you're saying.

Reply:
Not dumb! This one is tricky and worth discussing. The trade off would be that you have to wrap the inner text in the desired color. `<span class="has-text-gray-dark">Lorem ipsum dolor </span>`. You'll see that in the markup. I hesitated on that bc it's purely to compensate for our CSS flexibility - aka keep us from either a.) having border color helpers b.) building in a sponsor color variation for this util. I don't think b.) is a deal breaker by any means, so if that feels better since we're not polluting on the HTML side, I can totally do that.

Reply:
I think wrapping inner text with a desired color helper is just fine!